### PR TITLE
[LTC] Remove non-native view ops

### PR DESCRIPTION
**Pull request by alanwaketan**
Summary:
LTC somehow implements a bunch of non-native view ops during the transition to functionalization. Let's remove them now that functionalization is final.

Test Plan:
CI.

### CODE REVIEW

File: torch/csrc/lazy/ts_backend/ts_node_lowering.cpp [hunk 1]
```diff
@@ -158,192 +158,6 @@ torch::lazy::TSOpVector Scalar::Lower(
...
-}
-
-torch::lazy::TSOpVector Resize::Lower(
-    std::shared_ptr<torch::jit::GraphFunction> function,
-    torch::lazy::TSLoweringContext* loctx) const {
-  std::vector<torch::jit::NamedValue> arguments;
-  for (const torch::lazy::Output& output : operands()) {
-    arguments.emplace_back(loctx->GetOutputOp(output));
-  }
-  return LowerBuiltin(this, function, arguments);
```
Comment (JackCaoG) [inline torch/csrc/lazy/ts_backend/ts_node_lowering.cpp:293]: what's this LowerBuiltin? Should we remove this function too?

**Review by JackCaoG — APPROVED** [review]

File: torch/csrc/lazy/ts_backend/ts_node_lowering.cpp (hunk 1 above)
Comment (antoniojkim) [reply torch/csrc/lazy/ts_backend/ts_node_lowering.cpp:293]: It's defined above and still in use. So, we shouldn't remove it

**Review by antoniojkim — APPROVED** [review]

**Pull request closed without merging**